pages/common/batch.md

```diff
@@ -0,0 +1,16 @@
+# batch
+
+> Execute commands at a later time when the system load levels permit.
+> \*Make sure the atd (or atrun) is running.
```

Comment:
As it is not part of the description itself wanted to make feel of a note here.
Maybe should use parenthesis ?